pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -78,11 +78,11 @@ var (
 	OperatorVersionRHSSO               OperatorVersion = "8.0.1"
 	OperatorVersionRHSSOUser           OperatorVersion = "8.0.1"
 	OperatorVersionCodeReadyWorkspaces OperatorVersion = "2.0.0"
-	OperatorVersionFuse                OperatorVersion = "1.5.0"
+	OperatorVersionFuse                OperatorVersion = "7.5.0"
 	OperatorVersion3Scale              OperatorVersion = "0.4.0"
-	OperatorVersionCloudResources      OperatorVersion = "0.13.0"
+	OperatorVersionCloudResources      OperatorVersion = "0.13.1"
 	OperatorVersionUPS                 OperatorVersion = "0.5.0"
-	OperatorVersionApicurito           OperatorVersion = "1.5.0"
+	OperatorVersionApicurito           OperatorVersion = "7.5.0"
```

Comment:
Same with Fuse. The product version is `7.5.0` but the actual image tag we use for the operator is `1.5.0`